tox.ini

```diff
@@ -38,6 +47,12 @@ deps =
 [testenv:py34]
 basepython=python3.4
 
+[testenv:py34-with-django]
+basepython=python3.4
+deps =
+    -r{toxinidir}/requirements-dev.txt
+    django
```

Comment:
Then I'd prefer `django>=1.9.0,<1.10.0`.  I want the tests pinned to a specific whole number release of django.

You are also welcome to make a `with-django19` and a `with-django110` env to test both.